user-service/model/user-orm.js

```diff
@@ -42,7 +44,111 @@ export async function ormLoginUser(username, password) {
     }
 }
 
+export async function ormForgetPassword(username) {
+    // Make sure user exist
+    if (!await existsUser(username)) {
+        return { err: "User does not exist!"};
+    }
+
+    // create a One time link for 15 min 
+    const currPassword = await getPassword(username);
+    const secret = process.env.JWT_SECRET + currPassword;
+    const payload = {
+        username: username
+    };
+
+    const token = jwt.sign(payload, secret, {expiresIn: '15m'});

```

Comment:
Why are we signing the payload using JWT_SECRET + currPassword?

Reply:
It is to make sure it is uniqueness of the token.